src/components/WebexLocalMedia/WebexLocalMedia.js

```diff
@@ -16,30 +16,37 @@ import './WebexLocalMedia.scss';
  * @param {string} props.meetingID  ID of the meeting from which to obtain local media
  * @returns {Object} JSX of the component
  */
-export default function WebexLocalMedia({meetingID}) {
+export default function WebexLocalMedia({className, meetingID}) {
   const [mediaRef, {width}] = useElementDimensions();
   const {localVideo} = useMeeting(meetingID);
   const {ID} = useMe();
   const videoRef = useStream(localVideo);
 
-  const classBaseName = 'local-media';
-  const classObjects = {};
+  const classBaseName = `${WEBEX_COMPONENTS_CLASS_PREFIX}-local-media`;
+  const mainClasses = {
+    [classBaseName]: true,
+    desktop: width >= PHONE_LARGE,
+    'no-media': localVideo === null,
+  };
 
-  classObjects[`${WEBEX_COMPONENTS_CLASS_PREFIX}-${classBaseName}`] = true;
-  classObjects[`${WEBEX_COMPONENTS_CLASS_PREFIX}-${classBaseName}-desktop`] = width >= PHONE_LARGE;
-  classObjects[`${WEBEX_COMPONENTS_CLASS_PREFIX}-no-media`] = localVideo === null;
-
-  const cssClasses = classNames(classObjects);
+  if (className) {
+    mainClasses[className] = true;
+  }
 
   const disabledVideo = ID ? <WebexAvatar personID={ID} displayStatus={false} /> : <Spinner />;
 
   return (
-    <div ref={mediaRef} className={cssClasses}>
+    <div ref={mediaRef} className={classNames(mainClasses)}>
```

Comment:
Is there a reason you didn't invoke `classNames` at the const declaration?
The thing i liked about doing it that way is that is consistent with the way we do all other props, just passing an variable

Reply:
My reasoning on this was that we could inject some additional classes into `mainClasses` at a later point before the render. It also is the way it is done throughout the original widgets, so I carried it over.